.github/workflows/build-oss.yml

```diff
@@ -182,7 +182,7 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@7c2007bcb556501da015201bcba5aa14069b74e2 # 0.23.0
         with:
-          image-ref: nginx/nginx-ingress:${{ steps.meta.outputs.version }}
+          image-ref: ${{ steps.meta.outputs.tags }}
```

Comment:
Just curios, why was the `nginx/nginx-ingress` prefix removed as well in this change?

Reply:
It scans the image in GCR, not dockerhub